src/components/launcherwatchermodel.cpp

```diff
@@ -35,8 +35,13 @@ void LauncherWatcherModel::monitoredFileChanged(const QString &changedPath)
     if (!QFile(changedPath).exists()) {
         foreach (LauncherItem *item, *getList<LauncherItem>()) {
             if (item->filePath() == changedPath) {
-                removeItem(item);
-                emit filePathsChanged();
+                if (QFile::exists(changedPath)) {
```

Comment:
Is there a difference between QFile(path).exists() vs QFile::exists(path)? How can one be false and the other true?